src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -233,15 +233,20 @@ def load_builtin(
         tool: Tool,
         node_inputs: Optional[dict] = None,
     ) -> Tuple[Callable, dict]:
-        return BuiltinsManager._load_package_tool(tool.name, tool.module, tool.class_name, tool.function, node_inputs)
+        return BuiltinsManager._load_tool_from_module(
+            tool.name, tool.module, tool.class_name, tool.function, node_inputs
+        )
 
     @staticmethod
-    def _load_package_tool(tool_name, module_name, class_name, method_name, node_inputs: Mapping[str, InputAssignment]):
-        """Load package in tool with given import path and node inputs."""
-        m = importlib.import_module(module_name)
+    def _load_tool_from_module(
```

Comment:
I guess we may have two methods
`_load_tool_from_module` which only accepts a python module object
`_load_package_tool` which only  accepts module_name and calls `_load_tool_from_module`

The function interface is not that clean to have both `module_name` and `module`